newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -128,6 +130,41 @@ const formatValue = ({
     ? i18n._(t`Base layer`)
     : value;
 
+const InstructionMissing = (props: {|
+  instructionType: string,
+  isCondition: boolean,
+|}) => {
+  const { hasExtensionNamed } = React.useContext(ExtensionStoreContext);
+  const { name, behaviorName, extensionName } = getFunctionNameFromType(
+    props.instructionType
+  );
+  const extensionStoreMention = hasExtensionNamed(extensionName)
+    ? ' Try installing it from the extension store.'
+    : '';
+  if (behaviorName) {
+    return (
+      <span className="instruction-missing">
+        <span className="function-name">{name}</span>{' '}
+        {props.isCondition ? 'condition' : 'action'} on behavior{' '}
+        <span className="behavior-name">{behaviorName}</span> from{' '}
+        <span className="extension-name">{extensionName}</span> extension is
+        missing.
```

Comment:
Is there a way we can translate those parts easily?
Maybe having a `<Trans>{functionName} {type} on behavior {behavior} from {extension} extension is missing.</Trans>` ? Where each param is a span